cmd/config.go

```diff
@@ -245,8 +245,9 @@ func (d *ConfigDuration) UnmarshalYAML(unmarshal func(interface{}) error) error
 
 // GRPCClientConfig contains the information needed to talk to the gRPC service
 type GRPCClientConfig struct {
-	ServerAddress string
-	Timeout       ConfigDuration
+	ServerAddresses []string
+	ServerAddress   string
```

Comment:
Since this seems subtle, can we get in-depth descriptions of the distinction between ServerAddress and ServerAddresses, how they're handled differently, and when one would want to use each?